app/models/article.rb

```diff
@@ -1,5 +1,6 @@
 # frozen_string_literal: true
 
 class Article < ApplicationRecord
-  validates_presence_of :title, :body
+  validates_presence_of :title, :body, :category
+  enum category: %i[current sport local politics economy world entertainment]
```

Comment:
If category is mandatory there should probably be a catch-all like "Other".
I think best approach would be to default it to the catch-all when missing, instead of throwing error to frontend.

Reply:
current should be based on article creation date and should not be a category per se in backend